### ISSUE TITLE
Parameter format date and date-time break generation

### ISSUE THREAD

**Issue by nmcb** [zalando/play-swagger#19]
The parameter `format: date` and `date-time`annotation for parameter `type: string` breaks generation:

API definition:
```
---
swagger: '2.0'
x-api-first-package: controllers
info:
  version: 0.0.0
  title: Simple API Test
paths:
  /{bippy}:
    get:
      parameters:
        - in: path
          name: bippy
          type: string
          format: date
          # path parameter required as a workaround for issue 5 
          required: true
      responses:
        200:
          description: bippy response
# model definition required as a workaround for issue 18          
definitions:
  bop:
    properties:
      name:
        type: string
```

Generation:

```
[play-swagger-test] $ compile
[info] Updating {file:/Users/marco/Code/play-swagger-test/}root...
[info] Resolving jline#jline;2.12.1 ...
[info] Done updating.
[info] Compiling 17 Scala sources and 1 Java source to /Users/marco/Code/play-swagger-test/target/scala-2.11/classes...
[error] /Users/marco/Code/play-swagger-test/conf/bippy.yaml:0: not found: type Date
[error] /Users/marco/Code/play-swagger-test/conf/bippy.yaml:0: not found: type Date
[error] /Users/marco/Code/play-swagger-test/conf/bippy.yaml:0: not found: type Date
[error] /Users/marco/Code/play-swagger-test/conf/bippy.yaml:0: not found: type Date
[error] /Users/marco/Code/play-swagger-test/conf/bippy.yaml:0: not found: type Date
[error] /Users/marco/Code/play-swagger-test/conf/bippy.yaml:0: not found: type Date
[error] 6 errors found
[error] (compile:compileIncremental) Compilation failed
[error] Total time: 3 s, completed Oct 13, 2015 7:05:05 PM
```

**Comment by slavaschmidt** [comment]
Should be fixed in #42

**Comment by slavaschmidt** [comment]
In version 0.1.4 compilation fails because of the missing import for every generated component:

not found: type DateMidnight
